ci-operator/step-registry/azure/provision/bastionhost/azure-provision-bastionhost-commands.sh

```diff
@@ -333,9 +333,8 @@ vhd_data=${bastion_ignition_file}
 
 echo "Create a Storage Account for bastion vhd"
 # 'account_name' must have length less than 24, so hardcode the basion sa name
-#sa_name_prefix=$(echo "${bastion_name}" | sed 's/[-_]//g')
-#sa_name="${sa_name_prefix}sa"
-sa_name="${JOB_NAME_HASH}bastionsa"
+sa_name_prefix=$(echo "${NAMESPACE}" | sed "s/ci-op-//" | sed 's/[-_]//g')
```

Comment:
could you give a sample of "NAMESPACE" ?

Reply:
Captured from log - `ci-op-8yvctx18`